### PR TITLE
Use a proper React file import for the OpenCost logo

### PR DESCRIPTION
**Pull request by mattray**
Fixes https://github.com/backstage/backstage/issues/20333

Screenshot 2023-10-05 at 9 15 47 PM

### CODE REVIEW

**Comment by mattray** [comment]
It could probably be cherry-picked on to the v1.18 branch if there's going to be a v1.18.5

**Review by tudi2d — APPROVED** [review]
Looks good to me - small comments on the changeset & documentation. I think it is reasonable to go with a "minor" release here even though it is at `0.1` as the current minor version is broken.

> It could probably be cherry-picked on to the v1.18 branch if there's going to be a v1.18.5

The versions are mostly fo

**Comment by mattray** [comment]
@tudi2d changes made, thanks for the review!

File: .changeset/young-toes-knock.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-opencost': minor
```
[the remark this answers is not in the document]
Comment (mattray) [reply .changeset/young-toes-knock.md]: Updated accordingly.

**Review by benjdlambert — APPROVED** [review]

**Comment by benjdlambert** [comment]
Nice! thanks @mattray!

**Comment by Nayaksonal** [comment]
Thanks for fix

**Pull request merged**